### DISCUSSION TITLE
BItbucket Server Username/Password Authentication not working

### DISCUSSION THREAD

**Discussion by mattfirtion** [renovatebot/renovate#28445] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

NPM 37.301.4

### Please tell us more about your question or problem

Username and password authentication for Bitbucket Server no longer works after [#28093 ](https://github.com/renovatebot/renovate/pull/28093) was added.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by mattfirtion** [comment, marked as answer]
We had configured `--token` for our Github token to get release notes which was the issue since username, password, and token were all set. Using the GITHUB_COM_TOKEN environment variables resolved issue.

**Reply by viceice** [reply]
using token for the GitHub token was always wrong when using platforms other than GitHub
